relay-filter/src/browser_extensions.rs

```diff
@@ -77,6 +77,8 @@ lazy_static! {
         # Dragon Web Extension from Nuance Communications
         # See: https://forum.sentry.io/t/error-in-raven-js-plugin-setsuspendstate/481/
         plugin\.setSuspendState\sis\snot\sa\sfunction
```

Comment:
You need to add a `|` at the end of this line if you want to add a new regex

Reply:
Shoot, you're right. Thanks for catching that. Fixed.